pkgs/games/multimc/icon.svg

```diff
@@ -0,0 +1,353 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
It should not be necessary to store an icon in this package repository, especially if it is not considered FOSS. This should be fetched using a URL.

Reply:
Should get squashed/rebased when removed so we don't accidentally include an unfree asset in the commit history.

Reply:
I will update the package to download the icon instead of storing it in the repo. As for squashing, last time I contributed to nixpkgs (which was admittedly quite a long time ago), policy was to rebase on master and squash PRs into one commit before merging. Is that not still the case? If not, I will rebase such that the icon is not in the commit history.

Reply:
I'm not sure what the rules are but my recent PRs weren't squashed when merged.